content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,127 @@
+  * Differential drive, subscribes to message of type [Twist](http://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Twist.html).
+* URDF/XACRO (experimental).
+  
+
+For a "feel" of these features, see an [example project](https://github.com/o3de/RobotVacuumSample) which uses this Gem to run navigation stack.
+
+## Clone this repository
+
+Clone the gem repository from Github:
+
+```
+git clone https://github.com/o3de/o3de-extras
+```
+
+## Building
+
+The Gem is built through building a project which uses it. 
+Make sure to [source your ROS 2 workspace](https://docs.ros.org/en/humble/Tutorials/Configuring-ROS2-Environment.html#source-the-setup-files) before building.
+
+## Adding Gem to your project
+
+To use O3DE ROS 2 Gem in your project, you need to register the Gem with O3DE. First, clone this repository. Then,
+in O3DE folder:
+```
+scripts/o3de.sh register --gem-path <PATH_TO_CLONED_O3DE_EXTRAS>/Gems/ROS2
+scripts/o3de.sh enable-gem -gn ROS2 -pp <PATH_TO_YOUR_PROJECT>
+```
+
+## Example project
+
+You can test O3DE ROS 2 Gem with [this project](https://github.com/o3de/RobotVacuumSample).
+It will allow you to run robot navigation.
+All necessary assets are included.
+
+## User Guides
+
+Follow the [O3DE ROS2 Gem User Guide](ros2-gem-user-guide.md) to understand its concepts and components.
+For additional developer's documentation, check the [Gem documentation](https://github.com/o3de/o3de-extras/blob/development/Gems/ROS2/docs/guides/development_in_clion.md) and
+[Open 3D Engine Contributor guide](/docs/contributing)
+
+## How to create your own robotic simulation
+
+>This section is to be detailed.
+
+Once you are set up and familiar with the example project, consider the following steps:
+1. [Create a new O3DE project](/docs/welcome-guide/create/) 
+2. [Register ROS 2 Gem](#adding-gem-to-your-project) and other useful Gems to created project. 
+Follow [Registering Gems to a Project](/docs/user-guide/project-config/register-gems/) guide. 
+3. Create or import Assets for your robots and environment. 
+   1. You can use formats supported by O3DE.
+   2. You can import your robot from URDF/XACRO.
+   3. Imported models might require some adjustments to be simulation-ready.
+   4. Mobilize robot with Vehicle Dynamics controllers.
+4. Determine which sensors you need to simulate. 
+   1. Some sensors are already implemented in this Gem.
+      1. They might require specialization (implementation specific for particular models).
+      2. You might like to consider tradeoffs between performance and realism in each case.
+   2. Use `ROS2SensorComponent` as a base class if you are implementing a new sensor.
+5. Develop necessary sensors and their prefabs.
+6. Consider developing additional abstraction to handle spawning and despawning robots.
+   1. This would also be a valuable contribution to the Gem.
```

Comment:
suggestion: It might be worth linking to the contribution guide here potentially?

Reply:
Awesome thanks!